jax/extend/xla_bridge.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from jax._src.xla_bridge import (
+  register_backend_factory as register_backend_factory,
```

Comment:
I'm not 100% sure I want this API to be public, and very likely not under the current module name ("xla_bridge" is misleading if you're plugging a non-XLA backend into JAX).

I think the only public method should be `register_plugin`?